sourcetracker/_sourcetracker.py

```diff
@@ -478,22 +501,36 @@ def gibbs_sampler(sink, cp, restarts, draws_per_restart, burnin, delay):
         ordering (same ordering as `final_env_assignments`). The [i, j] entry
         is the environment that the taxon `final_env_assignments[i, j]` is
         determined to have come from in draw i (j is the environment).
-    """
+    times : np.array
```

Comment:
This change represents a backward-incompatible interface change, so should probably be noted in the `CHANGELOG.md` file.